uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/staff-leave-report.component.js

```diff
@@ -453,34 +465,24 @@ define([
 
     /**
      * Removes the given leave request from the section it currently belongs to
-     * (only the "approved", "pending", and "other" sections support request removal)
-     *
-     * If the leave request belonged to either the "approved" or "pending" section,
-     * then the numbers of the section will be recalculated
+     * and the number of their sections are recalculated.
```

Comment:
`Removes the given leave request from the section it currently belongs to and the number of their sections are recalculated` - not quite sure I understand the sentence. Could you please try to describe all in active voice?